File: ScpProject/DAL/DTOs/Metrics/completedMetricHistory.cs
using System;

namespace DAL.DTOs.Metrics
{
    public class CompletedMetricHistory
    {
        public int MetricId { get; set; }
        public string Name { get; set; }
        public double Value { get; set; }
        public DateTime CompletedDate { get; set; }
        public bool IsCompletedMetric { get; set; }
        public int Id { get; set; }
    }
}

File: ScpProject/StrengthConApp/src/app/Models/Metric/CompletedMetricHistory.ts
export class CompletedMetricHistory {
    public MetricId: number;
    public Name: string;
    public CompletedDate: Date;
    public Value: number;
    public IsSelected: boolean = false;
    public DisplayDate: NGXDisplayDate;
}
export class NGXDisplayDate {
    public startDate: Date;
    public endDate: Date;
}